branchinfo.txt

```diff
@@ -3,13 +3,9 @@
 # Each line is expected to be in the format "[Name]=[Value]".
 #
 #
-# !!! WARNING !!! WARNING !!! WARNING !!! WARNING !!! WARNING !!! WARNING
-#       "master" branch temporarily produced 1.0.1 builds. The value here
-#       should be > 1.0.1
-#
 MAJOR_VERSION=1
 MINOR_VERSION=0
-PATCH_VERSION=2
+PATCH_VERSION=1
```

Comment:
I assume we need to update the master branch as well? Otherwise we might conflict.